copter/source/docs/land-mode.rst

```diff
@@ -31,6 +31,12 @@ features:
     second.  It does not use the altitude to decide whether to shut off the
     motors except that the copter must also be below 10m above the home
     altitude.
+    
+.. note::
+    
+    Using a Weight on Wheels (WoW) switch will increase the decent rate and
```

Comment:
spelling  descent

Reply:
That's what I get for trying to work without a spell checker.